### DISCUSSION TITLE
Kurtosis Map

### DISCUSSION THREAD

**Discussion by barisgenc12** [dipy/dipy#2582] [Reconstruction]
Hello to everyone.
I have multishell diffusion images obtained with b=0, b=1000, b=2500 64 directions. When I want to extract the dipy and kurtosis map images after eddy correction on these images, the signal intensity is very low at the white matter and gray matter interfaces in the radial kurtosis maps.
What could be the reason for this?
Should I use an extra correction algorithm for diffusion kurtosis ? 

<img width="548" alt="Ekran Resmi 2022-04-25 17 08 45" src="https://user-images.githubusercontent.com/58102584/165106241-dcfda90a-c7e0-4289-b622-c4256608a856.png">

Dr. Barış

**Comment by ShreyasFadnavis** [comment]
Hi! You may want to run Patch2Self before doing DKI -- it's supposed to help with reducing the degenerate voxels from the model fit.

Using Patch2Self after Eddy is okay 😄 

Thanks,
Shreyas

**Reply by barisgenc12** [reply]
Thank you Shreyas.

While still zero voxel, it has decreased in image to the previous one.
<img width="535" alt="Ekran Resmi 2022-04-25 17 59 36" src="https://user-images.githubusercontent.com/58102584/165117316-4ac12063-2e1e-475a-af91-93497b7b1582.png">

**Reply by ShreyasFadnavis** [reply]
Yes! Is it possible for you to run Patch2Self before Eddy?

**Reply by barisgenc12** [reply]
I made eddy first. Then I did patch2self.
